pyuvdata/uvdata/uvfits.py

```diff
@@ -624,7 +624,24 @@ def read_uvfits(
             if "FRAME" in ant_hdu.header.keys():
                 xyz_telescope_frame = ant_hdu.header["FRAME"]
             else:
-                if "COMMENT" in self.extra_keywords:
+                # want to make a case-insensitive comparison so look for existance
```

Comment:
Can I suggest an alternative idea here? Rather than growing out this if/else decision tree, it seems like perhaps the default should just be ITRF, and if `FRAME` isn't set, then a message/warning issued indicating the use of said default value (otherwise one could imagine it growing larger and larger with time).

Reply:
That's definitely a reasonable solution for this too. Just reading the older code, I couldn't tell if we were trying to be flexible or rigid so I went with rigid.

Reply:
I think that's reasonable. This particular case is one that will hopefully go away -- it's caused by a new piece of software that @mkolopanis has already made an issue on.

Reply:
ITRF is the only actually supported one too. We could default the unknown case to "?????" as well. It is accepted as the "ignorant" case according to AIPS 117. But the problem I can see then is if LAT, LON, and ALT are not set the telescope location will never be set.

Reply:
Yeah, that causes lots of issues. So what we did in the past was default to "????" but then for formats we knew were supposed to be in ITRF we set it to that to avoid the errors, leading to the current situation. I think it's ok to just set it to ITRF with a warning message.

Reply:
I've changed to this approach in https://github.com/RadioAstronomySoftwareGroup/pyuvdata/pull/1086/commits/3d93f4d7f51b34cc0ec9b0f135546475f1f5b3a8